### PR TITLE
fix(SR13): fixed exception text reinterpreted as HTML

### PR DESCRIPTION
**Pull request by IgnatG**
### Changeling
- Fixed SR13: exception text being sent as part of the response

### CODE REVIEW

File: portal-api/src/v1/gef/controllers/files.controller.js [hunk 1]
```diff
@@ -64,38 +67,40 @@ exports.create = async (req, res) => {
...
+      }),
+    );
 
     const status = processedFiles.every((file) => !!file.error) ? 200 : 201;
 
     return res.status(status).send(processedFiles);
   } catch (err) {
     console.error(`Error uploading file(s): ${err}`);
-    return res.status(500).send(err.message);
+    return res.status(500).send();
```
Comment (abhi-markan) [inline portal-api/src/v1/gef/controllers/files.controller.js:103]: Error message needs to be sanitized please

**Review by abhi-markan — CHANGES_REQUESTED** [review]
We need to sanitize the error message

File: portal-api/src/v1/gef/controllers/files.controller.js (hunk 1 above)
Comment (IgnatG) [reply portal-api/src/v1/gef/controllers/files.controller.js]: @abhi-markan updated

**Review by abhi-markan — APPROVED** [review]

**Comment by abhi-markan** [comment]
Nice work. Failing API test to address please.

**Pull request merged**